e2e/cypress.json

```diff
@@ -11,5 +11,9 @@
   "ignoreTestFiles": ["**/__snapshots__/*", "**/__image_snapshots__/*"],
   "projectId": "ed7hed",
   "defaultCommandTimeout": 10000,
-  "taskTimeout": 60000
+  "taskTimeout": 60000,
+  "retries": {
+    "runMode": 3,
```

Comment:
More info on this: https://www.cypress.io/blog/2020/08/19/introducing-test-retries-in-cypress-5-0/